lib/react_on_rails/server_rendering_pool/ruby_embedded_java_script.rb

```
@@ -230,7 +230,9 @@
           result = nil
           begin
             result = JSON.parse(result_string)
+            binding.pry
           rescue JSON::ParserError => e
+            return { html: result_string }
             raise ReactOnRails::JsonParseError.new(parse_error: e, json: result_string)
```

rubocop

[warning] Unreachable code detected.

(Lint/UnreachableCode)